dbt_platform_helper/utils/versioning.py

```diff
@@ -137,23 +135,23 @@ def get_platform_helper_versions(
 
 
 # Validates the returned PlatformHelperVersionStatus and echos useful warnings
-# Should use IO provider
 # Could return ValidationMessages (warnings and errors) which are output elsewhere
 def _process_version_file_warnings(versions: PlatformHelperVersionStatus):
     messages = versions.warn()
+    io = ClickIOProvider()
```

Comment:
should we move this to the parameters as a default so we can inject mocks during testing